content/copilot/using-github-copilot/ai-models/using-claude-sonnet-in-github-copilot.md

```diff
@@ -22,12 +22,14 @@ redirect_from:
 * {% data variables.product.prodname_copilot_chat_short %} in {% data variables.product.prodname_vscode %}
 * {% data variables.product.prodname_copilot_chat_short %} in {% data variables.product.prodname_vs %} 2022 version 17.13 or later
 * Immersive mode in {% data variables.product.prodname_copilot_chat_short %} in {% data variables.product.github %}
+* {% data variables.product.prodname_copilot_chat_short %} in JetBrains IDEs
```

Comment:
Why don't you have a variable with the product name for JetBrains IDEs? It seems like it's used a lot in the code.